modules/presets/secure-node.nix

```
@@ -39,7 +39,9 @@ in {
     };
 
     services.liquidd = {
-      prune = 1000;
+      # With validatepegin liquidd verifies that a transaction sending BTC into
```

pico-nit, but this is one of the few places where comments are relevant for end-users, so every bit of conciseness helps.
```suggestion
      # Enable `validatepegin` to verify that a transaction sending BTC into
```